src/Compilers/Test/Utilities/CSharp/CSharpTestBase.cs

```diff
@@ -403,6 +403,19 @@ public static SyntaxTree ParseWithRoundTripCheck(string text, CSharpParseOptions
             return CreateCompilation(source, references, options, parseOptions, TargetFramework.Mscorlib40, assemblyName, sourceFileName);
         }
 
+        public static CSharpCompilation CreateCompilationWithDocumentationComments(
```

Comment:
We try to avoid creating new `CreateCompilationX` methods, they quickly balloon out of control and make @jaredpar's life hard. Please just use `CreateCompilation` with the appropriate options parameters.